code/modules/mob/living/simple_animal/hostile/bear.dm

```diff
@@ -99,8 +99,58 @@
 		to_chat(user, "<span class='info'>You strap the armor plating to [A] and sharpen [A.p_their()] claws with the nail filer. This was a great idea.</span>")
 		qdel(src)
 
+mob/living/simple_animal/hostile/bear/butter //The mighty companion to Cak. Several functions used from it.
+	name = "Terrygold"
+	icon_state = "butterbear"
+	icon_living = "butterbear"
+	icon_dead = "butterbear_dead"
+	desc = "I can't believe its not a bear!"
+	faction = list("neutral", "russian")
+	obj_damage = 11
+	melee_damage_lower = 1
+	melee_damage_upper = 1
+	armour_penetration = 0
+	response_harm = "takes a bite out of"
+	attacked_sound = 'sound/items/eatfood.ogg'
+	deathmessage = "loses its false life and collapses!"
+	butcher_results = list(/obj/item/reagent_containers/food/snacks/butter = 6, /obj/item/reagent_containers/food/snacks/meat/slab = 3, /obj/item/organ/brain = 1, /obj/item/organ/heart = 1)
+	attack_sound = 'sound/weapons/slap.ogg'
+	attacktext = "slaps"
+
+/mob/living/simple_animal/hostile/bear/butter/Life() //Heals butter bear really fast when he takes damage.
+	if(stat)
+		return
+	if(health < maxHealth)
+		heal_overall_damage(10) //Fast life regen, makes it hard for you to get eaten to death.
+
+/mob/living/simple_animal/hostile/bear/butter/attack_hand(mob/living/L) //Borrowed code from Cak, feeds people if they hit you. More nutriment but less vitamin to represent BUTTER.
+	..()
+	if(L.a_intent == INTENT_HARM && L.reagents && !stat)
+		L.reagents.add_reagent(/datum/reagent/consumable/nutriment, 1)
+		L.reagents.add_reagent(/datum/reagent/consumable/nutriment/vitamin, 0.1)
 
-
+/mob/living/simple_animal/hostile/bear/butter/CheckParts(list/parts) //Borrowed code from Cak, allows the brain used to actually control the bear.
+	..()
+	var/obj/item/organ/brain/B = locate(/obj/item/organ/brain) in contents
+	if(!B || !B.brainmob || !B.brainmob.mind)
+		return
+	B.brainmob.mind.transfer_to(src)
+	to_chat(src, "<span class='big bold'>You are a butter bear!</span><b> You're a mostly harmless bear/butter hybrid that everyone loves. People can take bites out of you if they're hungry, but you regenerate health \
+	so quickly that it generally doesn't matter. You're remarkably resilient to any damage besides this and it's hard for you to really die at all. You should go around and bring happiness and \
+	free butter to the station!</b>")
+	var/new_name = stripped_input(src, "Enter your name, or press \"Cancel\" to stick with Kerrygold.", "Name Change")
+	if(new_name)
+		to_chat(src, "<span class='notice'>Your name is now <b>\"new_name\"</b>!</span>")
+		name = new_name
+
+mob/living/simple_animal/hostile/bear/butter/AttackingTarget() //Makes some attacks by the butter bear slip those who dare cross its path.
+	if(isliving(target))
+		var/mob/living/L = target
+		if(!(L.mobility_flags & MOBILITY_STAND))
+		else
```

Comment:
why not just.... `if(L.mobility_flags & MOBILITY_STAND))` instead?

Reply:
Uh, thats a good question. I asked how to tell if a mob is prone in the discord, and thats what I was given so I just used it; didn't give it too much thought. It _should_ work so I don't think its that big of an issue.

Reply:
This is really gross formatting though, and unless you have a good reason to have an empty `if` statement, it'd be more preferable for have cleaner, more easily readable code.

Reply:
Fair enough. I've changed it to just a check to see if they're standing, rather than an if-else to see if they aren't standing.